src/routes/firmware.ts

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import FirmwareManager from 'util/firmware';
+const router = Router();
+
+const firmwareManager = new FirmwareManager();
+
+router.get('/install', async (req, res) => {
+  const firmwareFile = req?.body?.fileName || '';
+  if(!firmwareFile) {
+    res.json({
+        output: "No firmware file provided",
```

Comment:
return `{ error: "No firmware file provided" }`
to keep consistency with the installation method